docs/gen_ref_pages.py

```diff
@@ -36,8 +35,11 @@
             print("::: " + identifier, file=fd)
 
         mkdocs_gen_files.set_edit_path(full_doc_path, path)
-    else:
-        skipped_paths.append(str(path))
 
 with mkdocs_gen_files.open("reference/SUMMARY.md", "w") as nav_file:
+    # Remove SUMMARY file from search
+    nav_file.writelines("---")
+    nav_file.writelines("search:")
+    nav_file.writelines("  exclude: true")
+    nav_file.writelines("---")
```

Comment:
What kind of sorcery is this? 😆

Reply:
Should have linked, sorry :) source: https://squidfunk.github.io/mkdocs-material/blog/2021/09/26/excluding-content-from-search/#excluding-pages